### DISCUSSION TITLE
serve --watch stops working when an exception occur on windows

### DISCUSSION THREAD

**Discussion by mkvcc** [adonisjs/core#2905] [Help]
Hi.
on windows server "serve --watch" command stop working when there is an exception. it just freeze.
could this be related to adonis? 
I tried powershell and cmd but no luck.
thanks.

**Comment by RomainLanz** [comment]
Hey @mkvcc! 👋 

Would you mind providing more information, like the exception thrown, your Node.js version, and how it looks inside your terminal?

I am using Windows 10 with AdonisJS and never encountered this type of issue.
